tests/test_color_changer.py

```diff
@@ -0,0 +1,57 @@
+"""ColorChangerクラスのテストコードを記述するモジュール.
+
+@author: kawanoichi
+"""
+
+import unittest
+import cv2
+import os
+from pathlib import Path
+import sys
+sys.path.append(str(Path(__file__).parent.parent))
+sys.path.append(str(Path(__file__).parent.parent / "camera_system"))
+from camera_system.color_changer import ColorChanger  # noqa
+
+
+class TestColorChanger(unittest.TestCase):
+    def test_color_changer(self):
+        cc = ColorChanger()
+        read_path = os.path.dirname(os.path.realpath(__file__)) + "/test_image.png"
+        save_path = os.path.dirname(os.path.realpath(__file__)) + "/result_test_image.png"
+        if os.path.exists(save_path):
+            os.remove(save_path)
+        img = cv2.imread(read_path)
+        # 6色変換のテスト
+        cc.change_color(img, save_path)
+        actual_exist = os.path.exists(save_path)
+        self.assertEqual(True, actual_exist)
```

Comment:
直接書き込むのはよくないから，`expected_exist`みたいなのを用意した方がいい
```suggestion
        self.assertEqual(expected_exist, actual_exist)
```
変数名的に，ここはファイルが存在してることの確認だよね？
19~27行らへんはもう少しコメントが欲しいかも